ghga_connector/cli.py

```diff
@@ -57,7 +58,13 @@ def failure(self, message: str):
 cli = typer.Typer()
 
 
-@cli.command()
+def toggle_command(env_var_name: str):
```

Comment:
I think just adding the following after the upload function should work:

```
if  strtobool(os.getenv(env_var_name) or "false"):
    cli.command()(upload)
```

Reply:
Fair point, I've changed it.